substrate/frame/custom-account/src/lib.rs

```diff
@@ -163,4 +163,12 @@ pub mod pallet {
 			Self::deposit_event(Event::Deposit { value: numeric_amount });
 		}
 	}
+
+	impl<T: Config<I>, I: 'static> SortedMembers<T::AccountId> for Pallet<T, I> {
```

Comment:
This trait is required by `EnsureSignedBy<CouncilAccount, AccountId>` that we use in `EnsureRootOrHalfSenateOrCustomCouncil` to allow Congress Custom Account to schedule stuff.